src/streams/SubscriptionsCard.js

```diff
@@ -51,7 +51,7 @@ class SubscriptionsCard extends PureComponent<Props> {
 
 export default connect((state: GlobalState, props) => ({
   narrow: props.narrow || [],
-  // Main scrren long longer conatin drawer,
+  // Main screen no longer contains drawer,
```

Comment:
I am not sure if that comment makes sense as part of the source code.
Maybe it did make sense as a commit message. I would prefer to completely remove it.

Reply:
Sure. Should be done in a follow-up commit?

Reply:
Sure, a spellchecking vs removing a comment sounds like a different type of work.